src/handlers/Chrome74.ts

```diff
@@ -721,6 +721,66 @@ export class Chrome74 extends HandlerInterface
 		await this._pc.setLocalDescription(answer);
 	}
 
+	async pauseReceiving(localId: string): Promise<void>
+	{
+		this._assertRecvDirection();
+
+		logger.debug('pauseReceiving() [localId:%s]', localId);
+
+		const transceiver = this._mapMidTransceiver.get(localId);
+
+		if (!transceiver)
+			throw new Error('associated RTCRtpTransceiver not found');
+
+		transceiver.direction = 'inactive';
+		
+		const offer = { type: 'offer', sdp: this._remoteSdp!.getSdp() };
```

Comment:
Are you sure this works? I'm pretty sure that `this._remoteSdp!.getSdp()` is gonna generate a SDP with `a=recvonly`. Wouldn't it invalidate previous `transceiver.direction = 'inactive'`?

Reply:
In this case the remoteSdp should be `a=sendonly` (because it is a consumer) and the localSdp (generated with createAnswer) will override it with `a=inactive` 

I re-tested it and this is the screenshot of the renegotiation in webrtc-internals:
![Screenshot 2021-09-22 at 12 59 47](https://user-images.githubusercontent.com/512252/134333178-e4acb253-52e1-4b69-8e04-44bee3b13cf3.png)